.prettierrc.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  ...require('@spotify/prettier-config'),
+  importOrder: [
+    '^react(.*)$',
+    '',
```

Comment:
The empty new lines should be removed

Reply:
They are added intentionally, see the docs here:

https://github.com/IanVS/prettier-plugin-sort-imports#3-add-spaces-between-import-groups